### ISSUE TITLE
Better error messages

### ISSUE THREAD

**Issue by Karinon** [FREVA-CLINT/freva-deployment#57]
Many of the error messages are either cryptic or not helpful at all. It would really help if at least the more frequent ones would get catched and handled properly with a more meaningful text.

To give you a few examples:
- Wrong SSH-Password: Would it be possible to catch the stacktrace and allow it to enter the password again? Preferably with posting the username (to remind the user of it in case there is a typo as well).

- I somehow fucked up my freva-map (no idea how), the error message was: 

```
freva-deployment - ERROR - Could not update server information {'message': 'Internal Server Error'}
```

This was not helpful at all. Could we at least enter the information, that the error happened while updating the server information in freva-map?

I will post more, when I encounter them

**Comment by antarcticrainforest** [comment]
The freva-map thingy is not important, I am thinking of just deleting this functionality. I presume no one is using it - @eelucio I am right, or are you using the freva-map command to interact with the services?

**Comment by eelucio** [comment]
we could totally get rid of it if it makes live easier, so far I only manage the services directly through the VMs themselves. I think I checked once just to try and that's all.

**Comment by antarcticrainforest** [comment]
Do you have other examples?

**Comment by antarcticrainforest** [comment]
Somewhere you mentioned that it was hard to read the error messages because they are poorly formatted.

I instructed ansible to format stdout in yaml inseatd of json. The result looks something like this:

```
TASK [Gathering Facts] *********************************************************
fatal: [freva-plugin-playground.cloud.dkrz.de]: UNREACHABLE! => changed=false 
  msg: 'Failed to connect to the host via ssh: ssh: connect to host freva-plugin-playground.cloud.dkrz.de port 22: Connection timed out'
  unreachable: true

PLAY RECAP *********************************************************************
freva-plugin-playground.cloud.dkrz.de : ok=0    changed=0    unreachable=1    failed=0    skipped=0    rescued=0    ignored=0

I think that's better.
```

**Comment by antarcticrainforest** [comment]
Addressed in #105
